section4/exercises/classMethods.js

```diff
@@ -28,13 +28,33 @@ Make sure to run the file with node in your command line.
 // that the proteins have been changed.
 
 class Burrito {
-  constructor(protein, base) {
+  constructor(protein, base, toppings) {
     this.protein = protein;
     this.base = base;
+    this.toppings = toppings;
   }
 
   // ADD CODE
+  changeProtein() {
```

Comment:
These methods could be called changeProteinToPork, and addSalsaToToppings... their range is very limited. Use some parameters to allow you to change/add more options.